app/screens/settings-screen/account/id.tsx

```diff
@@ -28,42 +42,49 @@ export const AccountId: React.FC = () => {
   }, [LL, accountId])
 
   return (
-    <View style={styles.accountId}>
-      <Text type="p2">{LL.AccountScreen.yourAccountId()}</Text>
-      <View style={styles.wrapper}>
-        <View style={styles.accIdWrapper}>
-          <View style={styles.accIdXs}>
-            {Array(20)
-              .fill(null)
-              .map((_, i) => (
-                <View style={styles.circle} key={i} />
-              ))}
+    <View>
+      <Text type="p2" bold>
+        {LL.AccountScreen.accountId()}
+      </Text>
+      {loading ? (
+        <Skeleton style={styles.wrapper} />
```

Comment:
I'm not sure a skeleton here would be the best UX. most of the data should be already available here in most case, and if not, shouldn't we already show the title of what is about the shown instead of a skeleton? 

skeleton feels more useful when the data has to be accurate from the beginning or it's not very determinisc what to expect. just my thought but we should ask opinion from more people on this.

Reply:
The account ID header is displayed irrespective of loading state or not. The skeleton and the container have same height so the content also just appears when the data is loaded. I can show you a GIF if you want.